rename `Query` to `Statement` (#1250)
scylladb/scylla-rust-driver: docs/source/data-types/data-types.md

See the following chapters for examples on how to send and receive each data type.

See [Statement values](../statements/values.md) for more information about sending values in queries.\

> See [Query result](../statements/result.md) for more information about reading values from queries

Reviewer: :wrench:

Reviewer: ```suggestion
See [Statement values](../statements/values.md) for more information about sending values along with statements.\
See [Query result](../statements/result.md) for more information about retrieving values from queries
```
